PyInstaller/utils/hooks/subproc/enchant_datafiles_finder.py

```diff
@@ -11,23 +11,24 @@
 import os
 import enchant
 
+
 def _win32_data_files():
     # This is basically a copy of enchant.utils.win32_data_files as of
     # release 1.6.0. We use this as a fallback for older versions of
     # enchant which do not have this function.
     # enchant is licenced under LGPL.
-    dataDirs = ("share/enchant/myspell","share/enchant/ispell","lib/enchant")
-    mainDir = os.path.abspath(os.path.dirname(enchant.__file__))
-    dataFiles = []
-    for dataDir in dataDirs:
+    data_dirs = ("share/enchant/myspell", "share/enchant/ispell", "lib/enchant")
```

Comment:
Could these relative 'myspell/ispell' directories also work when the right macports/homebrew prefix is supplied?
like: `data_dirs = ('BREW_PREFIX/share/enchant/myspell', 'PORTS_PREFIX/share/enchang/myspell')?